src/components/account/Account.js

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import Container from 'react-bootstrap/Container';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Spinner from 'react-bootstrap/Spinner';
+import { GrFormClose } from 'react-icons/gr';
+import { updateUser, resetUpdateUserState } from '../../redux/users/usersEdit';
+import { destroyUser, resetDestroyUserState } from '../../redux/users/usersDestroy';
+import { updateCurrentUser, signOut } from '../../redux/auth/auth';
+import defaultAvatar from '../../assets/images/profile-pic.png';
+
+const Account = () => {
+  const userState = useSelector((state) => state.users_edit);
+  const userDeleteState = useSelector((state) => state.users_destroy);
+  const { userSignedIn, currentUser } = useSelector((state) => state.auth);
+  const formInitialState = {
+    name: currentUser?.name,
+    email: currentUser?.email,
+    phone: currentUser?.phone,
+    password: '',
+    password_confirmation: '',
+    current_password: '',
+    avatar: {
+      value: '',
+      preview: currentUser?.avatar || defaultAvatar,
+      file: null,
+    },
+  };
```

Comment:
2. (a) Destructure the `currentUser` object in the `useSelector` hook for better readability.
    (b) Consider using optional chaining (?.) for all properties accessed on the `currentUser` object to avoid 
        `TypeErrors` in 
          case it is null or undefined.
    (c) Extract the `formInitialState` object into a separate constant to improve code modularity and readability.